src/site/_includes/macros/post-card.njk

```diff
@@ -1,51 +1,60 @@
-{% from 'macros/tag-chips.njk' import tagChips with context %}
+{% from 'macros/tag-pills.njk' import tagPills with context %}
 
 {#
-  @typedef {EleventyCollectionItem} post - The post to display in card format.
-  @property {boolean} [featured] - If a card should be a featured card.
-  @property {string} [className] - A space separated list of classes to apply.
+  @typedef {string} param.title - The title of the card, like a show's name or a post's title.
+  @typedef {string} param.subhead - A description or short summary.
+  @typedef {string} param.url - The url for the page the card is previewing.
+  @typedef {string|null} param.thumbnail - An relevant image to be displayed in the card.
+  @typedef {string|null} param.alt - Alt text for image. If no thumbnail is provided, this can be null.
+  @typedef {string[]|null} param.authors - An array of author ids for the authors who wrote the post.
+  @typedef {string[]|null} param.tags - An array of tag ids for the tags relevant to the post.
+  @typedef {Date|null} param.date - Date for when a post was created.
+  @typedef {Date|null} param.updated - Date for when a post was updated.
+  @typedef {boolean|null} param.isAuthor - Conditional if the card is for an author. This requires a different image.
```

Comment:
As you noted in the param description, w don't care if the card is for the author. We care only that it should get a rounded image. So I'd rename it to e.f. "image-avatar" or "image-round" or best "imageType=[round|square]".